libraries/session-storage/impl/src/main/sqldelight/io/element/android/libraries/matrix/session/SessionData.sq

```diff
@@ -1,13 +1,20 @@
+------------------------------------
+--  Current version of the DB: 4  --
+------------------------------------
+
 CREATE TABLE SessionData (
     userId TEXT NOT NULL PRIMARY KEY,
     deviceId TEXT NOT NULL,
     accessToken TEXT NOT NULL,
     refreshToken TEXT,
     homeserverUrl TEXT NOT NULL,
     slidingSyncProxy TEXT,
+    -- added in version 2
     loginTimestamp INTEGER,
+    -- added in version 3
     oidcData TEXT,
-    isTokenValid INTEGER NOT NULL,
+    -- added in version 4
+    isTokenValid INTEGER NOT NULL DEFAULT 1,
```

Comment:
`DEFAULT 1` was missing here, and is present in the file `3.sqm`, which migrate the data.